pylate/evaluation/nano_beir_evaluator.py

```diff
@@ -0,0 +1,206 @@
+from __future__ import annotations
+
+import logging
+from typing import Literal
+
+from sentence_transformers.evaluation.SentenceEvaluator import SentenceEvaluator
+from sentence_transformers.util import is_datasets_available
+
+from .pylate_information_retrieval_evaluator import PyLateInformationRetrievalEvaluator
+
+logger = logging.getLogger(__name__)
+
+DatasetNameType = Literal[
+    "climatefever",
+    "dbpedia",
+    "fever",
+    "fiqa2018",
+    "hotpotqa",
+    "msmarco",
+    "nfcorpus",
+    "nq",
+    "quoraretrieval",
+    "scidocs",
+    "arguana",
+    "scifact",
+    "touche2020",
+]
+
+
+dataset_name_to_id = {
+    "climatefever": "zeta-alpha-ai/NanoClimateFEVER",
+    "dbpedia": "zeta-alpha-ai/NanoDBPedia",
+    "fever": "zeta-alpha-ai/NanoFEVER",
+    "fiqa2018": "zeta-alpha-ai/NanoFiQA2018",
+    "hotpotqa": "zeta-alpha-ai/NanoHotpotQA",
+    "msmarco": "zeta-alpha-ai/NanoMSMARCO",
+    "nfcorpus": "zeta-alpha-ai/NanoNFCorpus",
+    "nq": "zeta-alpha-ai/NanoNQ",
+    "quoraretrieval": "zeta-alpha-ai/NanoQuoraRetrieval",
+    "scidocs": "zeta-alpha-ai/NanoSCIDOCS",
+    "arguana": "zeta-alpha-ai/NanoArguAna",
+    "scifact": "zeta-alpha-ai/NanoSciFact",
+    "touche2020": "zeta-alpha-ai/NanoTouche2020",
+}
+
+dataset_name_to_human_readable = {
```

Comment:
You can rename this to

MAPPING_DATASET_NAME_TO_READABLE_FORMAT

with uppercase